### ISSUE TITLE
Validate Tutanota mail addresses when adding a notification mail address

### ISSUE THREAD

**Issue by johnbotris** [tutao/tutanota#3296]
from a TODO

https://github.com/tutao/tutanota/blob/6e244133786874af29a8b58158faccab18873ab5/src/settings/IdentifierListViewer.js#L221-L225

It looks like the intent is that we want to check if the Tutanota mail address exists, and probably show an error if it doesn't. Would be a good idea

**Comment by bedhub** [comment]
Yes, we can check if the email address is valid. We could use `isTutanotaMailAddress` from RecipientInfo.js and then do `resolveRecipientInfo` do check if the address is active but this will not cover the case of custom domain addresses that are hosted at Tutanota. 

We could do `resolveRecipientInfo` first and then if the type is  RecipientInfoType.External and  `isTutanotaMailAddress` is true show the error message.
